django_quickbooks/processors/base.py

```diff
@@ -6,34 +6,36 @@
 from django_quickbooks import QBXML_RESPONSE_STATUS_CODES
 from django_quickbooks.decorators import realm_connection
 from django_quickbooks.exceptions import QBXMLParseError, QBXMLStatusError
+from django_quickbooks import utils
 
 
 class ResponseProcessor:
     resource = None
     op_type = None
     obj_class = None
 
-    def __init__(self, response, hresult, message):
+    def __init__(self, response, hresult, message, realm):
```

Comment:
You have to rebase your branch from `master` because master branch is diverted, and I came to conclusion that I do not have to store realm attribute inside ResponseProcessor, but instead, I have to pass it as an argument while processing ResponseProcessor. What do you think about it?

Reply:
I agree, I found that you `_process` upon initialization of the object, so I had to pass `realm` in `__init__`. 
can you show me code on how to achieve that?

BTW I believe this branch is rebased but from `master`, however I am still learning git and github.